docs/6_jupyterhub.md

```diff
@@ -168,7 +223,7 @@ If your processes require more than these limits, reach out to the <a href="http
 Finally, you can also edit text files like R scripts directly on JupyterHub. Clicking on Text File icon will open a new file that you can edit. Similarly, clicking on Python File will create an empty `.py` file and clicking on R File will create an empty `.r` file.
 You can also navigate to a directory that has the scripts you want to edit and double click on the script name to open it up in the Text Editor.
 
-For example, navigate to `rf_bootcamp_2024/examples/r_examples` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:
+For example, navigate to `intro_to_yens/examples/` folder in file brower first then double click on `investment-npv-parallel.R` file to open it in the text editor:
```

Comment:
the git repo is "intro_to_yens_2024"

Reply:
Oh shoot I dont think the change pushed my apologies.